user/language-clients.md

```diff
@@ -45,9 +46,64 @@ func main() {
 	if reflect.TypeOf(res.UAST).Name() != "Node" {
 		fmt.Errorf("Node must be the root of a UAST")
 	}
+
+	query := "//*[@roleIdentifier and not(@roleQualified)]"
+	nodes, _ := tools.Filter(res.UAST, query)
+	for _, n := range nodes {
+		fmt.Println(n)
+	}
 }
 ```
 
-Currently we're integrating [`libuast`](https://github.com/bblfsh/libuast) into
-the clients which will greatly increase the capabilities of the clients adding
-[XPath](https://en.wikipedia.org/wiki/XPath)-like querying of the UAST tree.
+## Query language
+
+[`libuast`](https://github.com/bblfsh/libuast) provides a query language for UASTs.
```

Comment:
"That provides a `XPath-like` query language" maybe?

It mentions XML below but without that context people could wonder why we're mapping things to a XML node.

Reply:
right, should provide a reference to xpath too